src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/StandardizedCompetencyDTO.java

```diff
@@ -1,29 +1,17 @@
 package de.tum.in.www1.artemis.web.rest.dto.competency;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 import de.tum.in.www1.artemis.domain.competency.CompetencyTaxonomy;
 import de.tum.in.www1.artemis.domain.competency.StandardizedCompetency;
 
 /**
- * DTO containing {@link StandardizedCompetency} data. It only contains the id of the knowledge area and source.
+ * DTO used to send requests regarding {@link StandardizedCompetency} objects.
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-@JsonIgnoreProperties(ignoreUnknown = true)
-public record StandardizedCompetencyDTO(Long id, String title, String description, CompetencyTaxonomy taxonomy, String version, Long knowledgeAreaId, Long sourceId) {
-
-    /**
-     * Creates a StandardizedCompetencyDTO from the given StandardizedCompetency
-     *
-     * @param competency the StandardizedCompetency
-     * @return the created StandardizedCompetencyDTO
-     */
-    public static StandardizedCompetencyDTO of(StandardizedCompetency competency) {
-        Long sourceId = competency.getSource() == null ? null : competency.getSource().getId();
-        Long knowledgeAreaId = competency.getKnowledgeArea() == null ? null : competency.getKnowledgeArea().getId();
-
-        return new StandardizedCompetencyDTO(competency.getId(), competency.getTitle(), competency.getDescription(), competency.getTaxonomy(), competency.getVersion(),
-                knowledgeAreaId, sourceId);
-    }
+public record StandardizedCompetencyDTO(@NotNull @Size(min = 1, max = StandardizedCompetency.MAX_TITLE_LENGTH) String title,
```

Comment:
If you only use it for requests, I would name it accordingly so you can easily find it in the autocompletion.
Same for the other DTO